analyser/module_analyser_call.c2

```diff
@@ -247,7 +247,7 @@ fn bool on_format_specifier(void* context, printf_utils.Specifier specifier, u32
         Expr* arg = args[fa.idx];
         QualType qt = arg.getType();
         qt = qt.getCanonicalType();
-        if (!qt.isInt32()) {
+        if (!qt.promotesToInt32()) {
```

Comment:
Is this needed for the raw strings?

Reply:
not really related to rawstrings, it is a fix for `printf` argument checking.  I can split it into a separate PR.